src/amo/components/SearchPage/index.js

```diff
@@ -1,18 +1,60 @@
+/* @flow */
+import React from 'react';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
 
 import Search from 'amo/components/Search';
-import { convertQueryParamsToFilters } from 'core/searchUtils';
+import {
+  convertFiltersToQueryParams,
+  convertQueryParamsToFilters,
+} from 'core/searchUtils';
 
 
-export function mapStateToProps(state, ownProps) {
+type PropTypes = {|
+  filters: Object,
```

Comment:
Since you're opting for Flow and this already has a type definition, you might as well hook it up. It will give you protection around using unknown filters.

```diff
diff --git a/src/amo/components/SearchPage/index.js b/src/amo/components/SearchPage/index.js
index 82df0709..21b6b50d 100644
--- a/src/amo/components/SearchPage/index.js
+++ b/src/amo/components/SearchPage/index.js
@@ -1,4 +1,5 @@
 /* @flow */
+/* global $PropertyType */
 import React from 'react';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
@@ -8,10 +9,11 @@ import {
   convertFiltersToQueryParams,
   convertQueryParamsToFilters,
 } from 'core/searchUtils';
+import type { SearchParams } from 'core/api/search';
 
 
 type PropTypes = {|
-  filters: Object,
+  filters: $PropertyType<SearchParams, 'filters'>,
   pathname: string,
 |};
 
diff --git a/src/core/api/search.js b/src/core/api/search.js
index 737e154b..b5cfd0da 100644
--- a/src/core/api/search.js
+++ b/src/core/api/search.js
@@ -12,7 +12,7 @@ import type { ApiStateType } from 'core/reducers/api';
 import { convertFiltersToQueryParams } from 'core/searchUtils';
 
 
-type SearchParams = {|
+export type SearchParams = {|
   api: ApiStateType,
   auth: boolean,
   // TODO: Make a "searchFilters" type because these are the same args
```